JobRecruiting/JobRecruiting/src/main/java/com/recruit/jobrecruiting/entity/Skill.java

```diff
@@ -1,68 +0,0 @@
-/*
- * To change this license header, choose License Headers in Project Properties.
- * To change this template file, choose Tools | Templates
- * and open the template in the editor.
- */
-package com.recruit.jobrecruiting.entity;
-
-import java.io.Serializable;
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
-
-/**
- *
- * @author DENISA
- */
-@Entity
-public class Skill implements Serializable {
```

Comment:
nu ar fi bine oare sa avem totusi o enumeratie?

Reply:
Ne-am gandit ca pentru inceput sa facem field-ul skills la JobPost de tipul String simplu, sa nu ne mai complicam, ca in documentatia aia de la prof nu scrie clar sub ce forma vrea skills-urile. De-aia am sters entitatea aia de Skill. Sau te referi la altceva? Gen sa folosim un enum Skill pentru field-ul skills de la JobPost?

Reply:
ma referea sa avem o lista de skills, dar skill sa fie enum - adica mna... valori prefefinite gen - ca poate vrei sa le afisezi frumos si sa nu te apuci sa faci cu split si dealea ca sa faci rost de o lista

Reply:
Aaa ok, am inteles. Eu l-am pus String simplu sa ne fie mai usor sa lucram cu el. Dar sunt de acord cu ideea ta. Poti modifica tu?

Reply:
sure

Reply:
totusi mi se pare mai complicat cu lista de enums, nu merge cu many to many ca nu e entitate, e enum, si e nevoie de mai multe skills acolo (si cred ca am ajunge la un toString , adica nu am rezolva nimic)
ar fi mai simplu cum a fost initial, un tabel, asa isi mai poate adauga skiluri fara sa umble la cod
Opinions?